src/Collection.js

```diff
@@ -753,6 +803,68 @@ Collection.prototype.scroll = function (scrollId, options, filters, cb) {
 };
 
 /**
+ * Retrieves next result of a search with scroll query.
+ *
+ * @param {string} scrollId
+ * @param {object} [options] - Optional parameters
+ * @param {responseCallback} cb - Handles the query response
+ */
+Collection.prototype.scrollSpecifications = function (scrollId, options, cb) {
+  var
+    data = { scrollId: scrollId };
+
+  if (!scrollId) {
+    throw new Error('Collection.scrollSpecifications: scrollId is required');
+  }
+
+  if (!cb && typeof options === 'function') {
+    cb = options;
+    options = {};
+  }
+
+  this.kuzzle.callbackRequired('Collection.scrollSpecifications', cb);
+
+  if (options && options.scroll) {
+    data.scroll = options.scroll;
+  }
+
+  this.kuzzle.query(
+    { controller: 'collection', action: 'scrollSpecifications'},
+    this.kuzzle.addHeaders(data, this.headers),
+    options,
+    function (err, res) {
+      cb (err, err ? undefined : res.result);
+    }
+  );
+};
+
+/**
+ * Searches specifications across indexes/collections according to the provided filters
+ *
+ * @param {object} [filters] - Optional filters in ElasticSearch Query DSL format
+ * @param {object} [options] - Optional parameters
+ * @param {responseCallback} cb - Handles the query response
+ */
+Collection.prototype.searchSpecifications = function (filters, options, cb) {
```

Comment:
Other search methods return a KuzzleSearchResult object, shouldn't this one do the same?